static/coffee/controllers.coffee

```diff
@@ -485,7 +485,7 @@ controllers.controller('HomeController', ['$scope', '$controller', 'LabelService
         $scope.labels = labels
       )
     else if tab == 'users'
-      UserService.fetchNonPartner(true).then((users) ->
+      UserService.fetchAll(true).then((users) ->
```

Comment:
Can we change this to a control which indicates which users are being displayed without the user having to click on it?

It isn't obvious that a user should click something that just says "Select" to change the view of users.